docs/crudrice.livemd

```diff
@@ -216,6 +219,174 @@ root_sheafs
 
 ```
 
+## Sheaf Lattice Definition
```

Comment:
To get the meat of this PR, referring to the livebook from this section onwards should be sufficient.